### PR TITLE
Siamesex

### PR DESCRIPTION
**Pull request by Phancisth**
new files for detection mode added

### CODE REVIEW

File: detection mode/call_siamese.sh [hunk 1]
```diff
@@ -1,15 +1,18 @@
 #!/bin/sh
-#receiving of variable arguments to be revised (how to get inputPath and outputPath from DON's part)
+# This shell must receive the path of where the GitHub is cloned and stored in $cloneRepoPath
```
[the remark this answers is not in the document]
Comment (cragkhit) [reply detection mode/call_siamese.sh]: Test

[the remark this answers is not in the document]
Comment (Phancisth) [reply detection mode/call_siamese.sh]: Please check the source code again. I've updated it using a similar technique as you suggested.

File: detection mode/call_siamese.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+#
+# Command-line arguments
+# $1 - path to the cloned repository
+# $2 - path to the idiom/non-idiom snippets
+# $3 - path for the output files to be saved
+# $4 - name for the cloned repository Elasticsearch index
+# $5 - commitID
+# $6 - version tracking num (start from 1) of the cloned repo
+
+idiomPath = "D:\senior-project\python3-idiom"
```
Comment (cragkhit) [inline detection mode/call_siamese.sh]: This must be parameterised.

File: detection mode/call_siamese.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+# Reading of $indexName variable is optional if Don's script can get ES index's name from the user already
+echo "Enter index name for your repository"
+read indexName
```
Comment (cragkhit) [inline detection mode/call_siamese.sh]: This must also be parameterised.

File: detection mode/call_siamese.sh (hunk 2 above)
Comment (Phancisth) [reply detection mode/call_siamese.sh]: Fixed, use `$2` instead

File: detection mode/call_siamese.sh (hunk 3 above)
Comment (Phancisth) [reply detection mode/call_siamese.sh]: Fixed, used `$4` instead

**Comment by Phancisth** [comment]
New output file naming pattern goes as follow: <index_name>_<commitTracking>_<commitID>.csv

**Pull request merged**